velox/exec/Window.cpp

```diff
@@ -141,6 +148,22 @@ void Window::addInput(RowVectorPtr input) {
   numRows_ += input->size();
 }
 
+void Window::reclaim(
+    uint64_t targetBytes,
+    memory::MemoryReclaimer::Stats& stats) {
+  if (nonReclaimableSection_) {
+    ++stats.numNonReclaimableAttempts;
+    return;
+  }
+
+  if (noMoreInput_) {
+    // TODO Add support for spilling after noMoreInput().
```

Comment:
Lets add an error message here.

Reply:
This is not an error.

Reply:
Ah I see what you mean, this is best effort and nothing can be done by Window at this point.

Just for my knowledge, if we reach a point where a single Window partition is very big and is holding up everything, then what's the failure path ? At some point Memory arbitration should just fail this query I imagine.

Reply:
Yes, if we do run out of memory and cannot reclaim it from some operator, the query will fail.